guests.json

```diff
@@ -117,4 +117,8 @@
     "gif": "cursedparrot.gif",
     "name": "Cursed Parrot"
   }
+  {
```

Comment:
you are missing a , before this

Reply:
Oh!thank you!
I'll Try the request again.